docs/src/examples/examples_maps.md

```diff
@@ -3,23 +3,32 @@
 ```@example
 using VegaLite, VegaDatasets
 
-us10m = dataset("us-10m").path
-unemployment = dataset("unemployment.tsv").path
+#browsers don't open local files, so instead of providing a filepath to VegaLite we provide the data inline
+
+#us10m = dataset("us-10m").path
+us10m = read(dataset("us-10m"),String);
+#unemployment = dataset("unemployment.tsv").path
+unemployment = read(dataset("unemployment.tsv").path,String);
```

Comment:
I think with the new VegaDataset branch this whole thing could just look like `us10 = dataset("us-10m")` and the equivalent for unemployment. If that works, we also don't need the comment about local browsers.

Also, generally, feel free to just delete the old version of the code, no need to keep that around.

Reply:
Doesn't work now (VegaDatasets v2.0.0)
The us-10m dataset is fine, I guess because it is json format.
But reading `unemployment` as
`unemployment = dataset("unemployment")`
is working but the data can not be parsed by VegaLite (I guess), or I don't know how to code it syntactically proper. I tried
```
data={
       values=unemployment,
       format={
           type=:tsv
       }
},
```
with different types.

Providing the raw data with
`unemployment = read(dataset("unemployment").path,String);`
still works find.

Reply:
Same thing for `dataset("airports")`, which is csv format.

Reply:
It seems to be not the case that json format works, and others (tsc,csv) do not work.
The example "U.S. State Capitals Overlayed on a Map of U.S." is not working, despite that only json data is used:

Not Working:
```
using VegaLite, VegaDatasets

us10m = dataset("us-10m")
capitals = dataset("us-state-capitals")

@vlplot(width=800, height=500) +
@vlplot(
    mark={
        :geoshape,
        fill=:lightgray,
        stroke=:white
    },
    data={
        values=us10m,
        format={
            type=:topojson,
            feature=:states
        }
    },
    projection={type=:albersUsa},
) +
@vlplot(
    :circle,
    data={
        values=capitals,
        format={
            type=:json
        }
    },
    longitude="lon:q",
    latitude="lat:q",
    color={value=:orange}
) +
@vlplot(
    mark={
        type=:text,
        dy=-10
    },
    data={
        values=capitals,
        format={
            type=:json
        }
    },
    longitude="lon:q",
    latitude="lat:q",
    text={
        field=:city,
        type=:nominal
    }
)

```
There seems to be a difference between the two datasets:
```
julia> dataset("us-10m")
Vega JSON Dataset

julia> dataset("us-state-capitals")
50x4 Vega dataset
lon      │ lat     │ state       │ city
─────────┼─────────┼─────────────┼────────────
-86.2791 │ 32.3615 │ Alabama     │ Montgomery
-134.42  │ 58.3019 │ Alaska      │ Juneau
-112.074 │ 33.4485 │ Arizona     │ Phoenix
-92.3311 │ 34.736  │ Arkansas    │ Little Rock
-121.469 │ 38.5556 │ California  │ Sacramento
-104.984 │ 39.7392 │ Colorado    │ Denver
-72.677  │ 41.767  │ Connecticut │ Hartford
-75.5268 │ 39.1619 │ Delaware    │ Dover
-84.2728 │ 30.4518 │ Florida     │ Tallahassee
-84.39   │ 33.76   │ Georgia     │ Atlanta
... with 40 more rows
```
Both are json format:
```
julia> us10m = read(dataset("us-10m").path,String);

julia> us10m[1:100]
"{\"type\":\"Topology\",\"transform\":{\"scale\":[0.003589294092944858,0.0005371535195261037],\"translate\":[-1"

julia> capitals = read(dataset("us-state-capitals").path,String);

julia> capitals[1:100]
"[\n  {\"lon\":-86.279118, \"lat\":32.361538, \"state\":\"Alabama\", \"city\":\"Montgomery\"},\n  {\"lon\":-134.41974"
```

Reply:
The London Tube Lines shows the same problem:
```
londonBoroughs = dataset("londonBoroughs")
londonCentroids = read(dataset("londonCentroids").path,String);
londonTubeLines = dataset("londonTubeLines")
```
works, but
```
londonBoroughs = dataset("londonBoroughs")
londonCentroids = dataset("londonCentroids")
londonTubeLines = dataset("londonTubeLines")
```
doesn't.
So, it seems, that only `topojson` works as intended.

Reply:
I think what should work is `data = unemployment`? I.e. skip the whole format, value etc. specifier?

Reply:
Yes, you are right. All are working as you suggested, except one:
Three Choropleths Representing Disjoint Data from the Same Table

This is the working version:
```
using VegaLite, VegaDatasets

us10m = dataset("us-10m")
rows = read(dataset("population_engineers_hurricanes").path,String);

@vlplot(
    description="the population per state, engineers per state, and hurricanes per state",
    repeat={
        row=["population", "engineers", "hurricanes"]
    },
    resolve={
        scale={
            color=:independent
        }
    },
    spec={
        width=500,
        height=300,
        data={
            values=rows,
            format={
                type=:csv
            }
        },
        transform=[{
            lookup=:id,
            from={
                data={
                    values=us10m,
                    format={
                        type=:topojson,
                        feature=:states
                    }
                },
                key=:id
            },
            as=:geo
        }],
        projection={type=:albersUsa},
        mark=:geoshape,
        encoding={
            shape={
                field=:geo,
                type=:geojson
            },
            color={
                field={repeat=:row},
                type=:quantitative
            }
        }
    }   
)
```
set
`data=rows,`
results in
`TypeError: e is undefined vega-lite.min.js:1:62162`

Any idea?

I have changed all other examples and could do a new PR if we resolve the last one above.

Reply:
Yes, I should really document the syntax for composite plots :) If you change this to this, things should work:
```julia
@vlplot(
    description="the population per state, engineers per state, and hurricanes per state",
    repeat={
        row=["population", "engineers", "hurricanes"]
    },
    resolve={
        scale={
            color=:independent
        }
    }
) +
@vlplot(
    width=500,
    height=300,
    data=dataset("population_engineers_hurricanes"),
    transform=[{
        lookup=:id,
        from={
            data={
                values=us10m,
                format={
                    type=:topojson,
                    feature=:states
                }
            },
            key=:id
        },
        as=:geo
    }],
    projection={type=:albersUsa},
    mark=:geoshape,
    shape={
        field=:geo,
        type=:geojson
    },
    color={
        field={repeat=:row},
        type=:quantitative
    }
)
```
Note how I also dropped the `encoding` sub-nesting.